routers/voice.py

```
@@ -0,0 +1,459 @@
+            }
+
+            try:
+                sentiment = requests.post(sentiment_url, headers = sentiment_headers, data = data, timeout = 8)
+                sentiment_human = json.loads(sentiment.text)
+
+                message_record = await messages_collections.find_one({'session_id': session_id})
+
+                for role in message_record['roles']:
+                    if not role['sentiment'] and role['type'] == 'human':
+                        role['sentiment'] = sentiment_human['sentiment'] 
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)